sn_node_manager/src/bin/cli/main.rs

```diff
@@ -278,17 +278,32 @@ pub enum SubCmd {
     },
     /// Start safenode service(s).
     ///
+    /// By default, the interval between starting each service is scaled automatically. The 'connection-timeout'
+    /// controls the maximum time to wait between starting each service. Use the 'interval' argument to set a fixed
+    /// interval and override the dynamic interval.
+    ///
```

Comment:
I haven't dug into the rest of the PR yet, but from the perspective of a node user it's not clear to me what 'dynamic interval' is just from these docs. `connection-timeout` is a limit, but how does the mechanism decide that dynamic interval?

E.g. what is the relation between connections and the dynamic interval?

Reply:
I might be wrong here, but I think the link between the two is that the connection timeout would define the maximum possible value for the dynamic interval. I think that should be communicated on both the start command itself and the argument.

I also agree that we need to explain what the dynamic interval is based on, which I believe is how quickly the previous node establishes connections to a close group. Although, to be honest, just like the previous measurement, which was the length of time it takes to establish an RPC connection, I'm not sure I understand why either of those things would have a bearing on how long we should wait to try and start the _next_ node. I think it would be useful if we explained that in the documentation.